iocage_lib/ioc_start.py

```diff
@@ -1167,3 +1268,183 @@ def find_bridge_mtu(self, bridge):
         ).split()
 
         return membermtu[5]
+
+    def _command(self, command, *args, fib=None, jail=None):
+        cmd = [command] + list(args)
+        if jail is not None:
+            cmd = ['jexec', jail] + cmd
+        if fib is not None:
+            cmd = ['setfib', fib] + cmd
+        iocage_lib.ioc_common.logit({
+            "level": "DEBUG",
+            "message": f"running command: {cmd}"
+        })
+        try:
+            output = iocage_lib.ioc_common.checkoutput(
+                cmd,
+                stderr=su.STDOUT
+            )
+        except su.CalledProcessError as err:
+            err.output = err.output.decode('utf-8')
+            raise
+
+        return output
+
+    # Wrapper functions enhance readability of long
+    # lists of command and make it easier to add future
+    # conditionals based on command args.
+    def __route__(self, *args, fib=None, jail=None):
+        return self._command('route', *args, fib=fib, jail=jail)
+
+    def __ifconfig__(self, *args, fib=None, jail=None):
+        return self._command('ifconfig', *args, fib=fib, jail=jail)
+
+    def __ngctl__(self, command, *args):
+        return self._command('ngctl', command, *args)
+
+    def get_ng_bridge_nextlink(self, bridge, start=0):
+        nextlink = start
+        while True:
+            try:
+                self.__ngctl__('msg', f"{bridge}:", 'getstats', f"{nextlink}")
```

Comment:
Again not very familiar with netgraph, but is it possible we might have loops because the user has perhaps his own setup included as well. So this loop never ends ? ( hope you don't mind my comments ;) Just want to be sure we are safe :) )